### DISCUSSION TITLE
Latest Patcher not patching Among Us? (1.4.0-ci.181)

### DISCUSSION THREAD

**Discussion by lbowers** [Impostor/Impostor#391] [Q&A] [answered]
Should the latest patcher (1.4.0-ci.181 Win64) be working with the latest Steam Among Us release? Whenever I start an online game, the region is "North America" and not "Impostor". It was working with previous releases.

**Comment by JohnTranQUT** [comment, marked as answer]
You should be able to go into the private server anyways, even though it says North America.

Its a known issue. Its to do with the Among Us Client (Not patcher). Believed to be a bug.
The patcher does work regardlessly, same with edits to region.json files.
https://github.com/Impostor/Impostor/issues/390

**Reply by lbowers** [reply]
Thanks for taking the time to answer! That's great news.
